plugins/modules/zos_tso_command.py

```diff
@@ -128,9 +146,14 @@ def copy_rexx_and_run_commands(script, commands, module):
         rc, stdout, stderr = module.run_command([tmp_file.name, command])
         command_results = {}
         command_results["command"] = command
+        command_results["origrc"] = rc
+        if rc <= max_rc:
+            rc = 0
+            command_results["failed"] = False
         command_results["rc"] = rc
         command_results["content"] = stdout.split("\n")
         command_results["lines"] = len(command_results.get("content", []))
+        command_results["stderror"] = stderr
```

Comment:
Good catch to include Standard error , although the correct response property is `stderr` not `stderror` , see Ansible docs [here](https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html#stderr)

Reply:
typo. fixed.